File: CS50/mario.py
# TODO


from cs50 import get_int


def main():
    height = get_height()
    for i in range(height):

       #code for spaces
       for m in range(height - i - 1,0,-1):
        print(" ", end="")

        #code for hashtag
       for h in range(i+1):
        print("#", end="")

        #code for middle space
       print("  ", end="")

        #code for hashtag
       for l in range(i+1):
         print("#", end="")

        #code for newline after every row
       print()


def get_height():
    while True:
        n = get_int("Height: ")
        if n > 0 and n < 9:
            return n

main()

File: CS50/mario.c
#include <cs50.h>
#include <stdio.h>


int main(void)
{
    //ensure input is valid
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);


    for (int i = 0; i < n; i++)
    {
        {
            //code for spaces
            for (int m = n - i -1; m > 0 ; m--)
            printf(" ");

            //code for hastags
            for (int h = 0 ; h <= i ; h++)
            printf("#");

            //code for space in between
            printf("  ");

            for (int h = 0 ; h <= i ; h++)
            printf("#");

        }

        // Move to next row
        printf("\n"); }

}

